backend/config/dev.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "from" : "[FROM EMAIL ID HERE]",
```

Comment:
from to etc are not config. Email service must accept them as parameters or else how is it scalable